CHANGELOG.md

```diff
@@ -1,26 +1,128 @@
-Change Log
-==========
+# Change Log
+
+## Version 3.1.0 *(2020-01-24)*
+
+This is a new minor release of Chucker. Please note that this minor release contains multiple new features (see below) as well as multiple bugfixes. 
+
+### Summary of Changes
+
+* The library is now fully converted to Kotlin!
+* The whole UI has been revamped to support Dark Theme.
+* The Response/Request Body is now displayed in a `RecyclerView`, drastically improving performances on big payloads.
+* HTTP Response/Request Body can now be saved on file.
+* Notifications for Throwable and HTTP Traffic are now going on two separate channels.
+* A lot of classes inside the `.internal` package have restricted visibility (from public to internal).
+
+### Wall of PRs
+
+Please find here the list of all the PRs that got merged inside the upcoming version.
+
+#### Enhancements
+
+* [#182] New: Notifications update
+* [#160] New: Add date formatting for requestDate and responseDate
+* [#156] New: Update assets to match changed design
+* [#145] New: Introduce a shared view model into transaction details activity + fragments
+* [#143] New: Feature/dark theme
+* [#138] New: Add an option to save the request or response body
+* [#119] New: Display explanation text on tabs if empty
+* [#118] New: Update the request/response body to use a monospace font
+* [#98]  New: Add search button on request tab
+
+#### Bugfixes
+
```

Comment:
Suggest to use some readable points for sections instead of PR names, since PR names are often quite incorrect about what they do.
```suggestion
* Fixed ANRs during big response payloads processing.
* Fixed contentType response formatting.
* Fixed notifications importance in Android Q.
* Fixed date formatting in transaction overview.
* Fixed visibility of internal library classes and resources.
* Fixed XML parsing
```